### DISCUSSION TITLE
doris BE在运行过程中，内存占用逐日增加，没有回落的迹象

### DISCUSSION THREAD

**Discussion by kychill1986** [apache/doris#17898] [A - General / Q&A]
doris在运行过程中，be内存占用逐日增加，没有回落的迹象，如图（3月13日重启过be）
![image](https://user-images.githubusercontent.com/2108750/225842207-daa32e77-eb98-44e3-a218-1f3d077dedb0.png)

相应的stream load current processing 也是一直在递增，总是感觉任务执行完后，资源没有被回收
![image](https://user-images.githubusercontent.com/2108750/225842074-99c86ce6-977e-43b3-be38-25915a115b0a.png)

如此这样后面会有OOM的风险，请大佬帮忙看下，是配置的问题，还是其他哪里有问题？

**Comment by LemonLiTree** [comment]


**Reply by maben996** [reply]
我这边在使用datax导入doris时，发现be会挂掉，可以帮忙看下这个问题的原因吗？
be.WARING日志：
<img width="1777" alt="image" src="https://github.com/apache/doris/assets/44827813/58e7fc25-9120-4b43-a7c7-19e13071938a">
但当我show tablet 时，发现tablet正常：
<img width="1780" alt="image" src="https://github.com/apache/doris/assets/44827813/9c07c6e9-ebe5-4015-a20e-79ee69b71480">
查看be.out日志：
<img width="1241" alt="image" src="https://github.com/apache/doris/assets/44827813/12c12aef-c4f3-401d-8271-23adc19ece0f">
解析coredump：
<img width="1771" alt="image" src="https://github.com/apache/doris/assets/44827813/fcc1219a-5ce5-4a1a-b21f-570b17fe2cf8">
